projects/swc-starter-webpack/src/index.js

```diff
@@ -32,10 +32,10 @@ import '@spectrum-web-components/popover/sp-popover.js';
 import '@spectrum-web-components/tooltip/sp-tooltip.js';
 import '@spectrum-web-components/picker-button/sp-picker-button.js';
 import '@spectrum-web-components/help-text/sp-help-text.js';
-import '@spectrum-web-components/menu/sp-menu.js';
-import '@spectrum-web-components/menu/sp-menu-item.js';
-import '@spectrum-web-components/menu/sp-menu-group.js';
-import '@spectrum-web-components/menu/sp-menu-divider.js';
+// import '@spectrum-web-components/menu/sp-menu.js';
+// import '@spectrum-web-components/menu/sp-menu-item.js';
+// import '@spectrum-web-components/menu/sp-menu-group.js';
```

Comment:
Can we remove them instead of commenting? Or we are saving it future PR?

Reply:
I have added them back, thought to save for future PR. Its fine to keep for now